backend/danswer/db/swap_index.py

```diff
@@ -67,6 +73,6 @@ def check_index_swap(db_session: Session) -> SearchSettings | None:
             for cc_pair in all_cc_pairs:
                 resync_cc_pair(cc_pair, db_session=db_session)
 
-            if MULTI_TENANT:
-                return now_old_search_settings
-    return None
+            old_search_settings = current_search_settings
+
+    return old_search_settings
```

Comment:
Should this be None? Otherwise, will reindex every time!

Reply:
it's defaulted to None at the start of the function now!